src/main/java/io/vertx/httpproxy/interceptors/impl/AddPrefixPathInterceptorImpl.java

```diff
@@ -11,10 +11,8 @@
 
 package io.vertx.httpproxy.interceptors.impl;
 
-import io.vertx.httpproxy.ProxyInterceptor;
-
-public class AddPrefixPathInterceptorImpl implements ProxyInterceptor {
-
-  // FIXME
-
+public class AddPrefixPathInterceptorImpl extends PathInterceptorImpl {
```

Comment:
Type inheritance isn't necessary here. You could have a single class with two constructors.